modules/govuk/manifests/node/s_publishing_api_db_admin.pp

```diff
@@ -0,0 +1,102 @@
+# == Class: Govuk_Node::S_publishing_api_db_admin
+#
+# This machine class is used to administer publishing_api PostgreSQL RDS instances.
+#
+# === Parameters
+#
+class govuk::node::s_publishing_api_db_admin(
+  $backup_s3_bucket     = undef,
+  $postgres_host        = undef,
+  $postgres_user        = undef,
+  $postgres_password    = undef,
+  $postgres_port        = '5432',
+  $postgres_backup_hour = 7,
+  $postgres_backup_min  = 30,
+  $apt_mirror_hostname,
+) {
+  include govuk_env_sync
+  include ::govuk::node::s_base
+
+  if $backup_s3_bucket {
+    $ensure = 'present'
+  } else {
+    $ensure = 'absent'
+  }
+
+  apt::source { 'gof3r':
+    ensure       => $ensure,
+    location     => "http://${apt_mirror_hostname}/gof3r",
+    release      => $::lsbdistcodename,
+    architecture => $::architecture,
+    key          => '3803E444EB0235822AA36A66EC5FE1A937E3ACBB',
+  }
+
+  package { 'gof3r':
+    ensure  => $ensure,
+    require => Apt::Source['gof3r'],
+  }
+
+  $alert_hostname = 'alert'
```

Comment:
Could you move this up to where the other variables are defined?

Reply:
This is done